crates/forge/tests/integration/signing.rs

```diff
@@ -3,10 +3,10 @@ use test_utils::running_tests::run_test_case;
 use test_utils::{assert_passed, test_case};
 
 #[test]
-fn simple_signing_flow() {
+fn simple_signing_flow_stark_curve() {
```

Comment:
Can we make the name of this test and behaviour it tests consistent with other curve tests?

Reply:
#1257